### ISSUE TITLE
org.apache.hadoop.hbase.client.RetriesExhaustedException: Can't get the locations

### ISSUE THREAD

**Issue by lonly197** [hortonworks-spark/shc#181]
**Params**
spark-submit  \
--conf "spark.ui.port=4042" \
--conf "spark.sql.codegen=true" \
--conf "hbase.zookeeper.quorum=dmp1,dmp2,dmp3" \
--conf "zookeeper.znode.parent=/hbase-unsecure" \
--class test.Example \
--master local[2] \
--name test \
--files /etc/hadoop/conf/core-site.xml,/etc/hbase/conf/hbase-site.xml \
--jars /usr/hdp/current/hbase-client/lib/htrace-core-3.1.0-incubating.jar,/usr/hdp/current/hbase-client/lib/hbase-client.jar,/usr/hdp/current/hbase-client/lib/hbase-common.jar,/usr/hdp/current/hbase-client/lib/hbase-server.jar,/usr/hdp/current/hbase-client/lib/guava-12.0.1.jar,/usr/hdp/current/hbase-client/lib/hbase-protocol.jar,/usr/hdp/current/hbase-client/lib/htrace-core-3.1.0-incubating.jar,/root/lonly/shc-core-1.1.1-2.1-s_2.11.jar \
/home/test/testshc_2.11-0.1.jar 

**Error**
![image](https://user-images.githubusercontent.com/7350356/30306645-f0f10a46-97ab-11e7-9b3e-509d3cffd255.png)

**Comment by weiqingy** [comment]
Maybe you want to try: copy  hbase-site.xml  to spark conf directory.

**Comment by weiqingy** [comment]
@thewilliamzhang  Yes. You can put hbase properties into a config file or json format. Then, as this example shows [here](https://github.com/hortonworks-spark/shc/blob/master/examples/src/main/scala/org/apache/spark/sql/execution/datasources/hbase/LRJobAccessing2Clusters.scala#L107), you can pass config file (HBaseRelation.HBASE_CONFIGFILE) or json (HBaseRelation.HBASE_CONFIGURATION) into `option()` function.

**Comment by weibingo** [comment]
@weiqingy  i will copy hbase-site.xml to spark conf directory, but don't work.
my env:  spark, hadoop, hbase also open kerberos 。 how i do it ?  
can give me your email ?
